wgsl/index.bs

```diff
@@ -3479,6 +3621,24 @@ Issue: Which index is used when it's out of bounds?
            (OpAccessChain)
 </table>
 
+<table class='data'>
+  <caption>Getting a pointer to a column vector from a pointer to a matrix</caption>
```

Comment:
we have no matrix access by letters (xyzw)?

Reply:
WGSL does not have it.
Similarly, MSL, GLSL don't have it. 
HLSL has two non-array-index syntaxes, for accessing a specific scalar component, but not for getting a (row or column) vector out of it. See https://docs.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-per-component-math#the-matrix-type

Reply:
Interesting. Shouldn't we then require the index to be a const expression?
SPIR-V has `OpVectorExtractDynamic` for vector indexing, but I'm not seeing anything about the matrix.

Reply:
It is not a strict requirement. If the matrix is in a variable then OpAccessChain would be used. If it is a constant value then if the index is a constant OpCompositeExtract could be used otherwise an implementation could generate a function scope variable to store the matrix and use an OpAccessChain to access it. Similar to array accesses, this aspect of SPIR-V is probably best handled by an implementation to allow a better user experience.

Reply:
I see. It's an unfortunate thing to work around, but your explanation makes sense